app-libs/libmspack/libmspack-0.5alpha.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="A portable library for Microsoft compression formats"
+DESCRIPTION="The following file formats are supported: \
+SZDD files, which use LZSS compression \
+KWAJ files, which use LZSS, LZSS+Huffman or deflate compression \
+.HLP (MS Help) files, which use LZSS compression \
+.CAB (MS Cabinet) files, which use deflate, LZX or Quantum compression \
+.CHM (HTML Help) files, which use LZX compression \
+.LIT (MS EBook) files, which use LZX compression and DES encryption \
+.LZX (Exchange Offline Addressbook) files, which use LZX compression \
+To determine the capabilities of the library, and the binary compatibility \
+version of any particular compressor or decompressor, use the \
+mspack_version() function. The UNIX library interface version is \
+defined as the highest-versioned library component."
+HOMEPAGE="https://www.cabextract.org.uk/libmspack/"
+COPYRIGHT="2000-2017 Stuart Caie"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/libmspack-$portVersion.tar.gz"
+CHECKSUM_SHA256="8967f275525f5067b364cee43b73e44d0433668c39f9376dfff19f653d1c8110"
+SOURCE_DIR="libmspack-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libmspack$secondaryArchSuffix = $portVersion
+	lib:libmspack$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libmspack${secondaryArchSuffix}_devel = $portVersion
+	devel:libmspack$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	libmspack$secondaryArchSuffix == $portVersion
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	prepareInstalledDevelLibs \
+		libmspack
```

Comment:
Please replace
```
	prepareInstalledDevelLibs \
		libmspack
```
by
```

	rm $libDir/libmspack.la

	prepareInstalledDevelLib libmspack

```
(keeping the blank lines, which make the recipe look nice ;-)